app/controllers/concerns/course/forum/topics_hide_concern.rb

```diff
@@ -0,0 +1,25 @@
+module Course::Forum::TopicsHideConcern
```

Comment:
@xzhflying I just I think it's because you didn't namespace the concern when include, this name will give you the same error